### PR TITLE
/projects 

### PR DESCRIPTION
**Pull request by Illustrova**
- Content rendering
- Project index page

### CODE REVIEW

**Comment by Illustrova** [comment]
fixed colors as discussed, ready for review

File: src/pages/projects/bem-project.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: BEM project
+date: 2016-12-01
```
Comment (varya) [inline src/pages/projects/bem-project.md:3]: 2013-08-31

File: src/pages/projects/hands-on-design-system-workshop.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Workshop “Hands on with a design system”
+date: 2019-11-01
```
Comment (varya) [inline src/pages/projects/hands-on-design-system-workshop.md:3]: 2018-06-01

File: src/pages/projects/huhtamaki.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Huhtamaki project
+date: 2019-01-01
```
Comment (varya) [inline src/pages/projects/huhtamaki.md:3]: 2018-05-31

File: src/pages/projects/react-finland-2029-workshop.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Workshop at React Finland 2020
+date: 2020-08-01
```
Comment (varya) [inline src/pages/projects/react-finland-2029-workshop.md:3]: 2020-05-25

**Review by varya — CHANGES_REQUESTED** [review]

**Pull request merged**